lib/mongo/collection/view/readable.rb

```diff
@@ -236,6 +236,12 @@ def estimated_document_count(opts = {})
             raise ArgumentError, "Cannot call estimated_document_count when querying with a filter"
           end
 
+          %w[limit skip].each do |opt|
```

Comment:
`%w` produces a list of strings, this needs to be `%i` to produce a list of symbols since the options are documented as being symbols.